src/BlazorUI/Bit.BlazorUI.Tests/SnackBar/BitSnackBarTests.cs

```diff
@@ -0,0 +1,205 @@
+            Assert.AreEqual(body, bodyElement.InnerHtml);
+        }
+    }
+
+    [DataTestMethod,
+        DataRow(true),
+        DataRow(false)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarAutoDismissTest(bool autoDismiss)
+    {
+        var com = RenderComponent<BitSnackBar>(
+            parameters =>
+            {
+                parameters.Add(p => p.AutoDismiss, autoDismiss);
+            }
+        );
+
+        await com.Instance.Show("title");
+
+        //Added a sec delay to be sure we are asserting after AutoDismissTime passed
+        await Task.Delay(com.Instance.AutoDismissTime + TimeSpan.FromSeconds(1));
+
+        var items = com.FindAll(".bit-snb-itm");
+
+        Assert.AreEqual(autoDismiss ? 0 : 1, items.Count);
+    }
+
+    [DataTestMethod,
+        DataRow(BitSnackBarType.Info),
+        DataRow(BitSnackBarType.Warning),
+        DataRow(BitSnackBarType.Success),
+        DataRow(BitSnackBarType.Error),
+        DataRow(BitSnackBarType.SevereWarning),
+        DataRow(null)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarTypeTest(BitSnackBarType? type)
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        if (type.HasValue)
+            await com.Instance.Show("title", string.Empty, type);
+        else
+            await com.Instance.Show("title");
+
+        var element = com.Find(".bit-snb-itm");
+
+        var typeClass = type switch
+        {
+            BitSnackBarType.Info => $"bit-snb-info",
+            BitSnackBarType.Warning => $"bit-snb-warning",
+            BitSnackBarType.Success => $"bit-snb-success",
+            BitSnackBarType.Error => $"bit-snb-error",
+            BitSnackBarType.SevereWarning => $"bit-snb-severe-warning",
+            _ => "bit-snb-info"
+        };
+
+        Assert.IsTrue(element.ClassList.Contains(typeClass));
+    }
+
+    [TestMethod]
+    public async Task BitSnackBarCloseButtonTest()
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        await com.Instance.Show("title");
+
+        var closeButton = com.Find(".bit-snb-cbt");
+        closeButton.Click();
+
+        var items = com.FindAll(".bit-snb-itm");
+
+        Assert.AreEqual(0, items.Count);
+    }
+
+    [TestMethod]
+    public async Task BitSnackBarDismissIconNameTest()
+    {
+        var com = RenderComponent<BitSnackBar>(
+            parameters =>
+            {
+                parameters.Add(x => x.DismissIconName, "Go");
+            }
+        );
+
+        await com.Instance.Show("title");
+
+        var closeButtonIcon = com.Find(".bit-snb-cbt > .bit-icon");
+
+        Assert.IsTrue(closeButtonIcon.ClassList.Contains("bit-icon--Go"));
+    }
+
+    [TestMethod]
+    public async Task BitSnackBarTitleTemplateTest()
+    {
+        var wrappedCom = RenderComponent<BitSnackBarWithTitleTemplate>();
+
+        var com = wrappedCom.FindComponent<BitSnackBar>();
+
+        await com.Instance.Show("title");
```

Comment:
again, plz change this expected value into the test method input argument.
also change that `second element` string into a parameter like Title, since it is also an expected value.